README.md

```diff
@@ -40,6 +40,138 @@ Then require the CSS in your `application.css` file:
 
 ## Usage
 
+### Rails >= 5.1
```

Comment:
I do not think we should go as far as recommend `form_with` for Rails 5.1. IMHO the `form_with` released in 5.1 is fundamentally broken as it does not include IDs. I personally would wait until 5.2 to make the leap from `form_for` to `form_with`.

Perhaps we could re-title this section with a caveat:

```
### `bootstrap_form_with` (Rails 5.1+)

Note that `form_with` in Rails 5.1 does not add IDs to form elements and labels by default, which are both important to Bootstrap markup. This behavior is corrected in Rails 5.2.
```

And place it *after* the section where we describe:

```
### `bootstrap_form_for` (Rails 5.0+)
```

Reply:
Yes. I wrote the README relatively early in my work on this PR, and once I actually started to use `form_with` in my own app I was feeling that `form_with` in 5.1 wasn't really ready for prime time.